package extra;

public class customException {

    //    public static void aMethod() throws Exception{
//        try {
//            throw new Exception();
//        }finally {
//            System.out.println("Finally");
//        }
//    }
    public static void main(String[] args) throws Exception {

//        try{
//            aMethod();
//        }catch (Exception e ){
//            System.out.println("exception");
//        }
//        System.out.println("finished");
//        try {
//            badMethod();
//            System.out.println("A");
//        }catch (RuntimeException exception){
//            System.out.println("B");
//        }catch (Exception ex1){
//            System.out.println("C");
//        }finally {
//            System.out.println("D");
//        }
//        System.out.println("E");
//    }
//    public static void badMethod(){
//        throw new RuntimeException();

    }
}
